medusa/post_processor.py

```diff
@@ -209,14 +198,14 @@ def recursive_glob(treeroot, pattern):
                     new_file_name = file_name
                     sub_file_name = file_name.rpartition('.')[0]
 
-                if is_subtitle and sub_file_name.lower() == base_name.lower().replace('[[]', '[').replace('[]]', ']'):
+                if is_subtitle and sub_file_name.lower() == base_name.lower():
```

Comment:
@medariox  need to find why this was added (blame) and if we can safely remove now